lib/cn/csched.c

```diff
@@ -69,6 +69,12 @@ csched_compact_status_get(struct csched *handle, struct hse_kvdb_compact_status
     sp3_compact_status_get(handle, status);
 }
 
+void
+csched_node_dirty(struct csched *handle, struct cn_tree_node *node)
+{
+    sp3_dirty_node(handle, node);
+}
+
```

Comment:
Unrelated, but this reminded me, can we remove file `csched_ops.h` ?

Reply:
`sp3_dirty_node` processes dirty nodes.  I think here all we want to do is mark the node dirty.  @beckerg ?  Maybe add an API to mark nodes dirty?

Reply:
Yeah, it's past time.  I should have done that way back when...

Agreed, then naming is misleading.  I'll open a ticket to rework this...